index.php

```diff
@@ -2,31 +2,34 @@
 <html lang="en">
```

Comment:
move to base.twig

so that all of your html is in your views

Reply:
just moved them over. I'll push them soon.